controller/transfer/association_transfer_manager.py

```diff
@@ -0,0 +1,111 @@
+import traceback
+from typing import Dict, Any
+from controller.labeling_task import manager as labeling_task_manager
+from controller.attribute import manager as attribute_manager
+from controller.record import manager as record_manager
+from util import notification
+
+
+from controller.information_source import manager as information_source_manager
+from submodules.model import enums
+from submodules.model.business_objects import (
+    general,
+    labeling_task_label,
+    record_label_association,
+)
+from submodules.model.models import RecordLabelAssociation
+from controller.weak_supervision import weak_supervision_service as weak_supervision
+
+
+def import_associations(
+    project_id: str,
+    user_id: str,
+    model_name: str,
+    labeling_task_name: str,
+    associations: Dict[str, Any],
+    indices: Dict[str, Any],
+    source_type: str,
+) -> int:
+    labeling_task = labeling_task_manager.get_labeling_task_by_name(
+        project_id, labeling_task_name
+    )
+
+    information_source = information_source_manager.get_information_source_by_name(
+        project_id, model_name
+    )
+    if information_source is None:
+        if source_type == "model_callback":
+            information_source = information_source_manager.create_information_source(
+                project_id,
+                user_id,
+                labeling_task.id,
+                model_name,
+                "",
+                "This is a model callback",
+                enums.LabelSource.MODEL_CALLBACK.value,
+            )
+        elif source_type == "heuristic":
+            information_source = information_source_manager.create_information_source(
+                project_id,
+                user_id,
+                labeling_task.id,
+                model_name,
+                "",
+                "This is a heuristic",
+                enums.LabelSource.INFORMATION_SOURCE.value,
+            )
```

Comment:
pretty similar maybe:
```suggestion
        
        if source_type == "model_callback":
            description = "This is a model callback"
            type = enums.LabelSource.MODEL_CALLBACK.value
        elif source_type == "heuristic":
            description = "This is a heuristic"
            type = enums.LabelSource.INFORMATION_SOURCE.value
            
        information_source = information_source_manager.create_information_source(
                project_id,
                user_id,
                labeling_task.id,
                model_name,
                "",
                description,
                type,
            )
```